specifications/move_adapter/README.md

```diff
@@ -866,135 +912,139 @@ has the following entries:
 * **_IDENTIFIERS_** - The set of identifiers in this binary. Identifiers are vectors of chars. Their format
 is the length of the vector in ULEB128 form followed by the chars. An identifier can only have characters in the
 ASCII set and specifically: must start with a letter or '\_', followed by a letter, '\_' or digit
-* **_ADDRESS_IDENTIFIERS_** - The set of addresses used in ModuleHandles. Addresses are fixed size so they are
+* **_ADDRESS\_IDENTIFIERS_** - The set of addresses used in ModuleHandles. Addresses are fixed size so they are
 stored contiguously in this table
-* **_STRUCT_DEFINITIONS_** - The structs or user types defined in the binary. A struct definition contains
+* **_STRUCT\_DEFINITIONS_** - The structs or user types defined in the binary. A struct definition contains
 the following fields:
-    * **_struct_handle_**: ULEB128 - index in the STRUCT_HANDLES table for the handle of this definition
-    * **_field_information_**: Field Information provides information about the fields of the struct or
+    * **_struct\_handle_**: ULEB128 - index in the STRUCT\_HANDLES table for the handle of this definition
+    * **_field\_information_**: Field Information provides information about the fields of the struct or
     whether the struct is native
         * **_tag_**: 1 byte - `0x1` if the struct is native, `0x2` if the struct contains fields, in which
         case it is followed by
         * **_field count_**: ULEB128 - number of fields for this struct
         * **_fields_**: a field count of
             * **_name_**: ULEB128 - index in the IDENTIFIERS table containing the name of the field
             * **_field type_**: [SignatureToken](#SignatureTokens) - the type of the field
-* **_STRUCT_DEF_INSTANTIATIONS_** - the set of instantiation for any given generic struct. It contains the following
+* **_STRUCT\_DEF\_INSTANTIATIONS_** - the set of instantiation for any given generic struct. It contains the following
 fields:
-    * **_struct handle_**: ULEB128 - index into the STRUCT_HANDLES table of the generic struct for
+    * **_struct handle_**: ULEB128 - index into the STRUCT\_HANDLES table of the generic struct for
     this instantiation (e.g. `struct X<T>`)
     * **_instantiation_**: ULEB128 - index into the SIGNATURES table for the instantiation of the struct
     The instantiation can be either partial or complete (e.g. `X<U64>` or `X<Z>` when inside another generic
     function or generic struct with type parameter `Z`)
-* **_FUNCTION_DEFINITIONS_** - the set of functions defined in this binary. A function definition contains the
+* **_FUNCTION\_DEFINITIONS_** - the set of functions defined in this binary. A function definition contains the
 following fields:
-    * **_function_handle_**: ULEB128 - index in the FUNCTION_HANDLES table for the handle of this definition
+    * **_function\_handle_**: ULEB128 - index in the FUNCTION\_HANDLES table for the handle of this definition
```

Comment:
wow to all of those....

Reply:
If you put them in backquotes (as code), then you don't generally need to escape the underscores. I assumed that you chose to write them this way on purpose, though